shell/browser/api/electron_api_web_contents.h

```diff
@@ -200,6 +200,8 @@ class WebContents : public ExclusiveAccessContext,
   void CloseDevTools();
   bool IsDevToolsOpened();
   bool IsDevToolsFocused();
+  std::string GetDevToolsTitle();
+  void SetDevToolsTitle(const std::string& title);
```

Comment:
Lots of the `UTF8ToUTF16` and `UTF16ToUTF8` can be removed if the API just takes `std::u16string`.